lib/core-integration/src/Test/Integration/Scenario/CLI/Transactions.hs

```diff
@@ -923,29 +911,7 @@ spec = do
                     , expectCliListItemFieldEqual 0 status InLedger
                     ]
```

Comment:
I wonder if we should keep the test. Since the point of the test was to forget the transaction, if that can't be done then... what's the point :thinking: ?

Reply:
we want to pass in linux and windows. in linux we need to wait for this and then try to forget, in windows this check is unneeded. so in general, I guess it is needed

Reply:
on the other hand the second waiting is unneeded : 
```
906         eventually_ $ do                                                                                                                                                                  
907             (fromStdout <$> listTransactionsViaCLI @t ctx [wDestId])                                                                                                                      
908                 >>= expectValidJSON (Proxy @([ApiTransaction n]))                                                                                                                         
909                 >>= flip verify                                                                                                                                                           
910                     [ expectCliListItemFieldEqual 0 direction Incoming                                                                                                                    
911                     , expectCliListItemFieldEqual 0 status InLedger                                                                                                                       
912                     ] 
```
forgetting depends on source wallet state not destination wallet state.

Reply:
I mean, the point of the test was to try forgetting a pending transaction. But the problem is, how to get a transaction that stays pending long enough such that we can actually forget it :|

Reply:
corrected description and removed unneeded eventually : https://github.com/input-output-hk/cardano-wallet/pull/1033/commits/b77d0673000fe5a8e4f89b770cf7e41539358483

Reply:
yes, we need to think how to make sure pending stays long enough in any OS. I am thinking now to propose using toxy proxy to ensure that. Meaning, well defined delay between wallet - > jormungandr ... just need to think about

Reply:
In the meantime there is also `TRANS_DELETE_01` on API suite which is fine for Windows. Seems that cli adds additional time which makes the tx go inLedger by the time we want to forget it via CLI :man_shrugging:

Reply:
Right, the CLI on windows is probably much slower than on linux :/